sopel/coretasks.py

```diff
@@ -496,6 +496,11 @@ def _parse_modes(bot, args):
         return
 
     channel = bot.channels[channel_name]
+
+    # Unreal sometimes sends an extraneous trailing space. If we're short an
+    # arg, we'll find out later.
+    if args[-1] == "":
+        args.pop()
     # Our old MODE parsing code checked for empty args. This would be a good
     # place to re-implement that if necessary for a non-compliant IRCd, but for
```

Comment:
> This would be a good place to re-implement that if necessary for a non-compliant IRCd

That's what you're doing. Feel free to reword this comment if you want, perhaps to talk about "more checks" for empty args other than at the end.

Reply:
You did well with the last fixup earlier today. 😸 It's good as-is, once squashed so CI can run one last time.